libsolidity/interface/CompilerStack.cpp

```diff
@@ -1206,13 +1205,75 @@ Json CompilerStack::ethdebug(Contract const& _contract, bool _runtime) const
 	solAssert(m_stackState >= AnalysisSuccessful, "Analysis was not successful.");
 	solAssert(_contract.contract);
 	solUnimplementedAssert(!isExperimentalSolidity());
+	Json result = Json::object();
+	Json contract = Json::object();
+	contract["name"] = _contract.contract->name();
+	Json definition = Json::object();
+	definition["sources"]["id"] = static_cast<int>(sourceIndices()[_contract.contract->sourceUnitName()]);
+	contract["definition"] = definition;
+	result["contract"] = contract;
+	result["environment"] = _runtime ? "call" : "create";
+	result["instructions"] = ethdebugInstructions(_contract, _runtime);
+	return result;
```

Comment:
```suggestion
	Json result = 
    {
        { "contract", 
            {
                { "name", _contract.contract->name() },
                { "definition", { "sources", { "id", static_cast<int>(sourceIndices()[_contract.contract->sourceUnitName()])} } }
            }
        },
        { "environment ", _runtime ? "call" : "create" },
        { "instructions", ethdebugInstructions(_contract, _runtime) }
    };
	return result;
```

How about this? It's untested but I find this style much easier to parse.

Reply:
could also be used below, in the `ethdebugInstructions` implementation